### PR TITLE
Release v3.1.0-rc2

### PR DESCRIPTION
**Pull request by therealyingtong**

### CODE REVIEW

File: doc/release-notes/release-notes-3.1.0-rc2.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+The code preparations for the Canopy network upgrade are finished and included in this release. The following ZIPs are being deployed:
+
+- [ZIP 207: Funding Streams](https://zips.z.cash/zip-0207)
+- [ZIP 211: Disabling Addition of New Value to the Sprout Value Pool](https://zips.z.cash/zip-0211)
+- [ZIP 212: Allow Recipient to Derive Sapling Ephemeral Secret from Note Plaintext](https://zips.z.cash/zip-0212)
+- [ZIP 214: Consensus rules for a Zcash Development Fund](https://zips.z.cash/zip-0214)
+- [ZIP 215: Explicitly Defining and Modifying Ed25519 Validation Rules](https://zips.z.cash/zip-0215)
+
+Canopy will activate on testnet at height TODO, and can also be activated at a specific height in regtest mode by setting the config option `-nuparams=0xe9ff75a6:HEIGHT`.
```
Comment (daira) [inline doc/release-notes/release-notes-3.1.0-rc2.md:15]: Should we decide on the activation height now?

Comment (str4d) [reply doc/release-notes/release-notes-3.1.0-rc2.md:15]: It's still not set in the code until the final release, so choosing the activation height doesn't need to happen for the RC.

Comment (daira) [reply doc/release-notes/release-notes-3.1.0-rc2.md:15]: Also, it might be worth saying that rc2 does *not* enable Canopy support on testnet.

**Comment by daira** [comment]
@zkbot r+

**Pull request merged**